Mixer.Base/Services/InteractiveService.cs

```diff
@@ -76,6 +76,18 @@ public async Task<IEnumerable<InteractiveGameListingModel>> GetSharedInteractive
             return await this.GetPagedAsync<InteractiveGameListingModel>("interactive/games/shared?user=" + channel.userId);
         }
 
+        /// <summary>
+        /// Gets all of the games where the channel owner has editor permissions.
+        /// </summary>
+        /// <param name="channel">The channel to get games for</param>
+        /// <returns>The games that have editor perssions set for the Channel Owner</returns>
```

Comment:
Minor typo: The games that have editor **_perssions_** set for the Channel Owner